### PR TITLE
feat: add validation to wallet form

### PR DESCRIPTION
**Pull request by Yulatun**
### Type of change:

- [X] Feature (introduces new functionality)
- [ ] Bugfix
- [ ] Configuration
- [ ] Refactoring (doesn't change functionality, only the code)
- [ ] Style changes (doesn't affect functionality, only the look of the components)

### PR Checklist:

- [x] Synced up with latest from the main branch/Code is linted
- [x] Description of the changes has been added (Screenshots/Video)

https://user-images.githubusercontent.com/39521158/180176541-9653de34-b0be-4ea4-a3b5-f58b403ff880.mov
- [x] Chakra components/constants/hooks have been used wherever possible
- [x] Commit messages follow [conventional commits guideline](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#summary)
- [x] Console has been checked for warnings

#### Screenshots/Video before change
Add screenshots from the [current deployed state](https://sandbox-money-app.herokuapp.com/) of the changed components here.

https://user-images.githubusercontent.com/39521158/180177810-1f2bbbfa-623f-4f49-84d5-d704ade34e05.mov

#### Screenshots/Video after change
Add screenshots with the changes applied here.

### CODE REVIEW

File: src/components/AddWallet/AddWallet.jsx [hunk 1]
```diff
@@ -46,13 +46,23 @@ export const AddWallet = () => {
...
                 placeholder={i18next.t('addWallet.name.placeholder')}
               />
               <FormErrorMessage>
-                {errors.name && errors.name.message}
+                {errors.name?.type === 'required' &&
+                  'Please, enter wallet name'}
+                {errors.name?.type === 'minLength' &&
+                  'Name must be longer than 1 symbols'}
+                {errors.name?.type === 'maxLength' &&
+                  'Name must be no longer than 30 symbols'}
```
Comment (kalashnikau) [inline src/components/AddWallet/AddWallet.jsx:65]:
```suggestion
{() => {
   if(!error.name) return null;
  switch(errors.name?.type){
      case 'required': 
          return 'Please, enter wallet name';
          break;
     case 'maxLength': 
          return 'Name must be longer than 1 symbols';
          break;
     case 'minLength': 
          return 'Name must be no longer than 30 symbols';
          break;
  }
}}
```

but better move it to separate function

also error messages should be localised

Comment (Yulatun) [reply src/components/AddWallet/AddWallet.jsx]: can I don't use break? 
 cause the return statement unconditionally exits a block of code, any statements after them cannot be executed.

Comment (kalashnikau) [reply src/components/AddWallet/AddWallet.jsx]: yep, but just make sure return statement is there

Comment (Yulatun) [reply src/components/AddWallet/AddWallet.jsx]: great, and should i use a function with a currency error message too?

Comment (itsnotpolza) [reply src/components/AddWallet/AddWallet.jsx]: https://react-hook-form.com/api/useform/register
You can pass all of this to react-hook-form and let it do the validating.

Comment (Yulatun) [reply src/components/AddWallet/AddWallet.jsx]: Palina, I'm sorry, I don't get what you want me to change cause I thought I've already used react-hook-form for validation and somehow it validates input data.
My question was about the currency.error.message cause Anton made me change name.error.message, but not  currency.error.message

**Review by itsnotpolza — APPROVED** [review]

**Pull request merged**